modules/core/src/main/scala/org/scalasteward/core/application/Context.scala

```diff
@@ -77,12 +81,31 @@ final class Context[F[_]](implicit
 )
 
 object Context {
-  def step0[F[_]](config: Config)(implicit F: Async[F]): Resource[F, Context[F]] =
+
+  sealed trait StewardContext[F[_]] {
+    def runF: F[ExitCode]
+  }
+  object StewardContext {
+    final case class Regular[F[_]](context: Context[F]) extends StewardContext[F] {
+      override def runF: F[ExitCode] = context.stewardAlg.runF
+    }
+
+    final case class ValidateRepoConfig[F[_]](file: File)(implicit
+        val validateRepoConfigAlg: ValidateRepoConfigAlg[F],
+        val logger: Logger[F]
+    ) extends StewardContext[F] {
+      override def runF: F[ExitCode] = validateRepoConfigAlg.validateAndReport(file)
+    }
+  }
+
+  def step0[F[_]](
+      usage: Config.StewardUsage
+  )(implicit asyncF: Async[F]): Resource[F, StewardContext[F]] =
```

Comment:
Could you revert and keep original `(implicit F: Async[F])`?
It is a convention to declare `implicit F: ~[F]` in the existing code base.

Reply:
Yeah, sure, it's just that the pre-CI linter complained about it, I thought that was the issue somehow.